fiftyone/operators/executor.py

```diff
@@ -90,7 +90,7 @@ def to_json(self):
         }
 
 
-def execute_operator(operator_uri, ctx, params):
+def execute_operator(operator_uri, ctx):
```

Comment:
-1 for changing code in a doc PR. Can we move this to another PR?

Reply:
I partly reverted this change so that the syntax from 1.4 still works.

I do really want to encourage `execute_operator(operator_uri, ctx)` though for consistency with how `ctx` is defined in all other contexts. The syntax in 1.4 was 👎